packages/services/src/types/index.ts

```diff
@@ -28,13 +28,13 @@ export class ApiError extends Error {
   }
 }
 
-export type SpecificChainBalances = Partial<
-  Record<SpecificChain, Record<string, number>>
->;
+export type SpecificChainBalances = {
+  [K in SpecificChain]?: Record<string, number>;
+};
 
-export type SpecificChainTokens = Partial<
-  Record<SpecificChain, Record<string, string>>
->;
+export type SpecificChainTokens = {
+  [K in SpecificChain]?: Record<string, string>;
+};
```

Comment:
Had to adjust these to get the behavior I believe we want, which is "zero or more keys of SpecificChain, each referencing a defined Record<string, string>"